chain/rpc/src/lib.rs

```diff
@@ -297,6 +297,20 @@ impl<'a> IntoFuture for PendingTransaction<'a> {
     }
 }
 
+/// On-chain configuration Node Safe module
+#[derive(Clone, Debug, Copy, PartialEq, Eq)]
+pub struct NodeSafeModuleConfig {
```

Comment:
I'd would prefer to call it `NodeSafeModuleStatus`.

`Config` would suggest we want to configure it somehow, whereas this objects rather reflects the on-chain state of the module.

Reply:
Make sense. Will change it